api/odl.py

```diff
@@ -887,8 +878,13 @@ def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None, do_get=
             if terms:
                 concurrent_checkouts = subtag(terms[0], "odl:concurrent_checkouts")
                 expires = subtag(terms[0], "odl:expires")
+
                 if expires:
-                    expires = dateutil.parser.parse(expires)
+                    expires = util.datetime_helpers.to_utc(dateutil.parser.parse(expires))
+                    now = util.datetime_helpers.utc_now()
+
+                    if expires <= now:
+                            continue
```

Comment:
First, this makes sure `expires` is a timezone-aware datetime. Second, this skips any books that were already expired the first time we saw them.